src/components/Movies/Movies.js

```diff
@@ -2,9 +2,13 @@ import "./Movies.scss";
 import MovieCard from "../MovieCard/MovieCard";
 import StarRating from '../StarRating/StarRating'
 
-const Movies = ({ apiMovieData }) => {
+
+const Movies = ({ apiMovieData, handleMovieClick }) => {
+
   const movieCards = apiMovieData.map(movie => (
-    <div className="movie-card" key={movie.id}> 
+
+
+    <div onClick={() => handleMovieClick(movie.id)} className="movie-card" key={movie.id}> 
```

Comment:
One might consider creating a MovieCard component - but its entirely up to you! If the code complexity around these pieces starts to make you feel the need for a whole new component - go for it!